extensions/markdown-language-features/media/markdown.css

```diff
@@ -105,6 +105,18 @@ body.showEditorSelection li.code-line:hover:before {
 .vscode-high-contrast.showEditorSelection .code-line .code-line:hover:before {
 	border-left: none;
 }
+ul ul{
```

Comment:
[nit] Can you please merge these into a single rule separated by commas instead. It makes the intent of this rule more clear